Formula/sevenzip.rb

```diff
@@ -0,0 +1,36 @@
+class Sevenzip < Formula
+  desc "7-Zip is a file archiver with a high compression ratio"
+  homepage "https://7-zip.org"
+  url "https://7-zip.org/a/7z2106-src.7z"
+  version "21.06"
+  sha256 "675eaa90de3c6a3cd69f567bba4faaea309199ca75a6ad12bac731dcdae717ac"
+  license all_of: ["LGPL-2.1-or-later", "BSD-3-Clause"]
+
+  on_linux do
+    depends_on "gcc@9" => :build
```

Comment:
```suggestion
    depends_on "gcc@9"
```
Do you need GCC 9 specifically, and not any other version (except GCC 10)?

Reply:
Yes, this formula requires gcc 9 specifically, as gcc 10 adds more warnings that breaks the build.

Reply:
But I wasn't asking about GCC 10. What about GCC 5, 6, 7, or 8? Can't we disable the warnings?

Reply:
False alert, I've tested on my debian and all versions of gcc passed. It seems upstream fixed the build error with modern gcc.